AmplifyPlugins/API/Podfile

```diff
@@ -10,13 +10,15 @@ target 'AWSAPICategoryPlugin' do
 	pod 'Amplify', :path => '../../'
   pod 'AWSPluginsCore', :path => '../../'
   pod "Starscream", "~> 3.0.2"
+  pod "ReachabilitySwift", "~> 5.0.0"
```

Comment:
also need to update AmplifyPlugins.podspec for consumers if u haven't already
```
s.subspec 'AWSDataStorePlugin' do |ss|
    ss.source_files = 'AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/**/*.swift'
    ss.dependency 'AWSPluginsCore', AMPLIFY_VERSION
    ss.dependency 'SQLite.swift', '~> 0.12.0'
    ss.dependency 'ReachabilitySwift', '~> 5.0.0' // moved to APIPlugin
  end
```

Reply:
if you have a sample app, would be good to consume the pod again locally and see if it works

Reply:
Thanks!  Updated the pod spec, and compiled w/ a sample app to make sure i was able to consume the pod.